### DISCUSSION TITLE
JetPack 6/L4T 36.0 support

### DISCUSSION THREAD

**Discussion by robertogs2** [OE4T/meta-tegra#1348] [Q&A] [answered]
Hello, one question over here:
Do we have any ongoing effort or any plans to give support to JetPack 6 even on Early Access?
Any information regarding JP6 is appreciated
Thanks

**Comment by madisongh** [comment, marked as answer]
We certainly have plans, but we'll continue to rely on NVIDIA's public/GA releases for our layer.  Any early access release from NVIDIA would be under NDA.

**Reply by robertogs2** [reply]
Hi guys, following up with this conversation; we do have access to an EA, and I have been working with basic bring-up for Orin AGX successfully with a modified meta-tegra and a custom kernel recipe. Is there any way I can contribute to add the support so when this goes public there is already work done? 
I appreciate any suggestion, thanks!

**Reply by ichergui** [reply]
Hey @robertogs2 
We need to sync up with Nvidia team then we will get back to you.
Agree you need to wait until `Jetpack/L4T` be public and we are happy to get your contributions. I assume you have an EA under NDA, right ?

**Reply by robertogs2** [reply]
Hi, that sounds all right. In the meantime I can work on having my changes as close as meta-tegra wants if thats ok.
And yes, it is under NDA.
